src/icons/folderIcons.ts

```diff
@@ -456,7 +456,22 @@ export const folderIcons: FolderTheme[] = [
           'handlers',
         ],
       },
-      { name: 'folder-ansible', folderNames: ['ansible'] },
+      {
+        name: 'folder-ansible',
+        folderNames: [
+          'ansible',
+          'inventories',
+          'inventory',
+          'playbooks',
+          'collections',
+          'ansible_collections',
+          'roles',
```

Comment:
Not everyone who's using a 'roles' directory wants to see the Ansible folder icon. Please remove these generic names from the PR.